types/react-loadable/index.d.ts

```diff
@@ -182,7 +185,7 @@ declare namespace Loadable {
     }
 }
 
-declare const LoadableExport: Loadable.Loadable;
+declare const LoadableExport: LoadableExport.Loadable;
```

Comment:
I don't understand why just renaming the namespace would be sufficient for it to work 🤔

Reply:
because `LoadableExport` is a const name and a namespace name, few lines below - in the module declaration section - there is a statement `export = LoadableExport`

I'm not sure if this is not breaking change, because if peoples are using this package as a triple-slash reference the `Loadable` name might be broken, maybe the name should be `Loadable` instead of `LoadableExport`, how do you think?

Reply:
The change in name exports both meanings of LoadableExport which is const as well as namespace and lets user ability to use that to reference the interfaces in there. Changing name to Loadable wont be correct because Loadable is not  present at runtime. (since the name of the var exposed is LoadableExport..